app/models/video.rb

```diff
@@ -12,25 +12,4 @@ def self.search_by_title(search_term)
     Video.where('lower(title) LIKE ?', "%#{search_term}%").order('created_at DESC')
```

Comment:
I'm not sure whether this is really better because ```ILIKE``` seems to be a non-standard SQL whereas ```lower``` and ```LIKE``` are ANSI standard (I guess).